### PR TITLE
adding logging

### PR DESCRIPTION
**Pull request by bdlesk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -40,6 +40,12 @@
       <version>3.8.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
+   <!--   <type>bundle</type> -->
```
Comment (forestbelton) [inline pom.xml]: Can you remove the commented out line here?

File: src/main/java/com/github/forestbelton/glua/app/Glua.java [hunk 2]
```diff
@@ -11,15 +11,22 @@
 import org.jgrapht.graph.SimpleDirectedGraph;
 import org.jgrapht.traverse.TopologicalOrderIterator;
 
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+
 import java.io.File;
 import java.io.IOException;
 import java.io.PrintStream;
+//import java.util.logging.Logger;
```
Comment (forestbelton) [inline src/main/java/com/github/forestbelton/glua/app/Glua.java]: Let's remove all the commented out code here.

**Review by forestbelton — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (bdlesk) [reply pom.xml]: Do you want the entire line removed?  Or just the comment?

**Review by forestbelton — APPROVED** [review]

**Comment by forestbelton** [comment]
Looks great, thank you for your contribution!

**Pull request merged**